.github/workflows/package-size-main.yml

```
@@ -22,6 +22,10 @@ jobs:
         run: yarn install --immutable
       - name: build
         run: yarn build
+      - name: Remove Sourcemaps
+        run: |
+          rm -f dist/*.map
+          rm -f action/*.map
```

Should we use the `yarn clean:sourcemaps` script here?
```suggestion
          yarn clean:sourcemaps
```